packages/angular_devkit/build_angular/test/browser/differential_loading_spec_large.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Architect } from '@angular-devkit/architect/src/index2';
+import { join, normalize } from '@angular-devkit/core';
+import { updateJsonFile } from '../../../../../tests/legacy-cli/e2e/utils/project';
+import { browserBuild, createArchitect, host } from '../utils';
+
+
+describe('Browser Builder differential loading', () => {
+  const target = { project: 'app', target: 'build' };
+  let architect: Architect;
+
+  beforeEach(async () => {
+    await host.initialize().toPromise();
+    architect = (await createArchitect(host.root())).architect;
+  });
+
+  // Temporarily commented out
+  // afterEach(async () => host.restore().toPromise());
+
+  it('works', async () => {
+
+    const tsConfigJsonPath = join(host.root(), normalize('tsconfig.json'));
+
+    // HACK: Somehow there is currently an issue under Windows
+    const tsConfigJsonPathCorrected = tsConfigJsonPath.replace('/C/', 'C:/');
+
+    await updateJsonFile(tsConfigJsonPathCorrected, json => {
+      json.compilerOptions.target = 'ES2015';
+    });
+
+    await browserBuild(architect, host, target);
+
+    // Default files should be in outputPath.
+    expect(await host.scopedSync().exists(normalize('dist/runtime-es5.js'))).toBe(true);
```

Comment:
Maybe you can simplify this test by using the jasmine helps.

```ts
    expect(host.scopedSync().list(normalize('dist')))
      .toEqual(jasmine.arrayContaining([
        // list of files
      ]);

```

Reply:
I'll look into this tomorrow.

Reply:
I'd like to keep this for the time being, as the pattern used here is also used in an existing test. Is this ok?